webapp/src/reducers.js

```diff
@@ -22,6 +22,11 @@ const INITIAL_STATE = {
     data: window.localStorage ? window.localStorage.getItem('email') : ''
   },
 
+  shift: {
+    never: true,
```

Comment:
I don't get the difference between `never` and `pressed` 😅 
I see `shift` being used as a boolean on `ParcelsMap.js` and to me `if pressed` => `never = false`

am I missing something?

Reply:
:D